module4/index.md

```diff
@@ -16,15 +16,26 @@ In Module 4, students will begin to dive into the skills and mindsets necessary
 
 ## Curriculum
 
-< Under Construction >
-<!-- 
-
 ### Week 1
```

Comment:
Can you also add the intermission work in this Pr?

Reply:
Oh! Yes, definitely